Automated builds failing

It seems that all automated builds are now failing due to cLaTeXMath depending on parts of std::experimental::filesystem that are not found. I assume this is related to [this commit](https://github.com/NanoMichael/cLaTeXMath/commit/90c192eaa8b6958e1619939f65cb5a65088313a0). How should we fix this? Is this some kind of version mismatch issue that we can fix on this end, or should I just change the build scripts to pull a fixed version of cLaTeXMath that predates that change for now?

(Paging @sp1ritCS)

Nope, still failing. 
```
/usr/local/include/clatexmath/utils/string_utils.h:84:58: error: no matching function for call to ‘cref(<unresolved overloaded function type>)’
   s.erase(find_if(s.rbegin(), s.rend(), not1(cref(isspace))).base(), s.end());
```
(https://github.com/sp1ritCS/notekit/runs/1548543005?check_suite_focus=true)

this is probably the fault of https://github.com/NanoMichael/cLaTeXMath/commit/7332b189f11cbeee34a3fc3e45e8f3deae317d4f

summon @Xrysnow

--- a/mainwindow.cpp
+++ b/mainwindow.cpp
@@ -25,9 +25,9 @@ CMainWindow::CMainWindow(const Glib::RefPtr<Gtk::Application>& app) : Gtk::Appli
 	printf("\n");
 	
 	#ifdef HAVE_CLATEXMATH
-	LaTeX::init((data_path+"/data/latex").c_str());
+	tex::LaTeX::init((data_path+"/data/latex").c_str());
 	/* allow \newcommand to override quietly, since we will be rerendering \newcommands unpredictably */
-	TeXRender *r = LaTeX::parse(utf82wide("\\fatalIfCmdConflict{false}"),1,1,1,0);
+	tex::TeXRender *r = tex::LaTeX::parse(tex::utf82wide("\\fatalIfCmdConflict{false}"),1,1,1,0);
 	if(r) delete r;
 	#endif
 	

--- a/meson.build
+++ b/meson.build
@@ -12,7 +12,7 @@ deps = [
 	dependency('jsoncpp'),
 	dependency('zlib'),
 	dependency('fontconfig'),
-	dependency('clatexmath')
+	dependency('clatexmath', version: '>=0.0.3')
 ]
 
 

--- a/pkgs/flatpak/com.github.blackhole89.notekit.yaml
+++ b/pkgs/flatpak/com.github.blackhole89.notekit.yaml
@@ -109,7 +109,7 @@ modules:
     sources:
       - type: git
         url: https://github.com/NanoMichael/cLaTeXMath
-        commit: 81c9d00331ac615128ab39c0c5f0c21887cc2a78
+        commit: d8efee2c435fdb594910ee4c3248c74ab67c831b
         
   - name: notekit
     buildsystem: meson

